manoseimas/settings/base.py

```diff
@@ -157,6 +158,7 @@
     'manoseimas.compat',            # depends on: solutions
     'manoseimas.widget',
     'manoseimas.lobbyists',
+    'manoseimas.compatibility_test',
```

Comment:
You should add this to `manoseimas/settings/development.py` instead so you can still deploy the code without the webpack test page before this is merged.